tests/core/test_bungiesearch.py

```diff
@@ -81,7 +81,7 @@ def test_raw_fetch(self):
         Test searching and mapping.
         '''
         item = Article.objects.search.query('match', _all='Description')[:1:True]
-        self.assertTrue(hasattr(item, '_meta'), 'Fetching first raw results did not return an object with a _meta attribute.')
+        self.assertTrue(hasattr(item, 'meta'), 'Fetching first raw results did not return an object with a _meta attribute.')
```

Comment:
Minor: could you change the message from `(...) with a _meta attribute (...)` to `(...) with a meta attribute (...)`?